LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018-2020 Jean-Philippe Cugnet and Contributors
+Copyright (c) 2018-2021 Jean-Philippe Cugnet and Contributors
```

Comment:
Was there any code change in 2021? I usually bump this as part of my release procedure.

Reply:
> Was there any code change in 2021? I usually bump this as part of my release procedure.

IANAL. 
Since we've updated `mix.exs`, assuming that considered as code changes.
Also, not sure if date range if applicable in legal point of view as starting year should be enough?